plugin/pkg/admission/securitycontext/scdeny/admission.go

```diff
@@ -25,6 +25,13 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 )
 
+const (
+	SUPPLEMENTAL_GROUPS = "SupplementalGroups"
```

Comment:
BTW this is Java style for naming constants and Golang doesn't use it. In Golang we prefer camelCase and also be careful with a first letter -- when it's in upper case, it sets constant visibility to public.